drivers/clock_control/clock_control_esp32.c

```diff
@@ -126,17 +124,29 @@ static void esp32_clock_perip_init(void)
 #if CONFIG_SMP
 	rst_reas[1] = esp_rom_get_reset_reason(1);
 #endif
-
-	/* For reason that only reset CPU, do not disable the clocks
-	 * that have been enabled before reset.
-	 */
 	if ((rst_reas[0] == RESET_REASON_CPU0_MWDT0 || rst_reas[0] == RESET_REASON_CPU0_SW ||
```

Comment:
I think the comment at this place is needed, so maybe re-phrase the message so it is clear what is going on at this place.